time/doc.go

```diff
@@ -29,14 +29,19 @@ package from the go standard library.
           duration == duration = boolean
           duration < duration = booleans
       time
-        fields:
-          year int
-          month int
-          day int
-          hour int
-          minute int
-          second int
-          nanosecond int
+        functions:
+          year() int
+          month() int
+          day() int
+          hour() int
+          minute() int
+          second() int
+          nanosecond() int
+		  in_location(string) time
```

Comment:
Spacing on these doc strings is important and would normally need to be corrected, but in reviewing this code there are additional formatting that have nothing to do with this PR. I'll clean those up in a follow up. we use [outline](github.com/b5/outline) to convert these doc strings into HTML documentation.

Reply:
Sorry about that, looks like it might be a tabs vs spaces error on my part. I looked around in the repo for a script that generates the docs but couldn't find anything.